engine/core/inc/luos_list.h

```diff
@@ -92,7 +93,11 @@ typedef enum
     PARAMETERS, // depend on the service, can be : servo_parameters_t, imu_report_t, motor_mode_t
     ERROR_CMD,
 
+    // Retrocompatibility zone
     TEXT, // ASCII string
+    PRESSURE, // pressure_t (Pa)
+
+
```

Comment:
I think you should remove those useless spaces.

Reply:
Yes I will